src/module/item/ancestry/document.ts

```diff
@@ -175,8 +175,7 @@ class AncestryPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends
 
         for (const fieldName of ["hp", "speed", "reach"] as const) {
             if (changed.system[fieldName] !== undefined) {
-                const minimum = fieldName === "speed" ? 5 : 1;
-                const value = Math.floor(Math.clamp(Number(changed.system[fieldName]) || 0, minimum, 99));
+                const value = Math.floor(Math.clamp(Number(changed.system[fieldName]) || 0, 0, 99));
```

Comment:
This probably ought to be broken out more:
hp: increments of 2 between 0 and 12
speed: increments of 5 between 0 and 100
reach: same as speed

Reply:
Eventually a `NumberField` will take care of it for us!